viser/client/src/SceneTree.tsx

```diff
@@ -245,9 +246,7 @@ function SceneNodeAttributeHandler(props: {
     if (!props.obj.matrixWorldAutoUpdate) props.obj.updateMatrixWorld();
   }, [visibility, wxyz, position, props.obj]);
 
-  return (
-    <SceneNodeLabel visible={visibility && labelVisibility} text={props.name} />
-  );
+  return <SceneNodeLabel visible={labelVisibility} text={props.name} />;
```

Comment:
Label seems to appear even when scene node is hidden, can fix by changing the `labelVisibility` to `visibility && labelVisibility`

Reply:
Hmm I actually lightly prefer it if the label shows up when the scene node is hidden